scripts/processsourcefiles.sh

```diff
@@ -30,6 +30,10 @@ then
   mv content/en/docs content/en/development
   # DOCS BRANCHES
   echo '------ Cloning all docs releases ------'
+
+  # TODO(Evan): Avoid the need for `mv` above and make the below just ap
```

Comment:
I had to use `mv` because I couldnt figure out how to make "git clone" happy when trying to extract only the /docs/ folders out of each knative/docs archive branch (there are all the other files and folders in the repo that we dont want to end up in the site build).

Reply:
I clone the docs outside of `/content/en`, and then symlink deeper into the tree. Since the clone isn't in `/content/en`, it isn't published at all.